### ISSUE TITLE
Hamburger Menu opens when click is executed

### ISSUE THREAD

**Issue by schuettecarsten** [Windows-XAML/Template10#845]
The new HamburgerMenu is opened when user clicks on an item and Navigation to new page is executed. The Menu should not open on a symbol click that is not the current page.

**Comment by JerryNixon** [comment]
I am open to discussing this, but this behavior is by design. Clicking on the current button while collapsed will expand the menu to "explain" to the user why no navigation takes place.

**Comment by mvermef** [comment]
~~is that by design for splitview in general or is that a Template10 Burger default by design?~~
this is a design by default for Template10, other examples of splitview usage aren't this way... Groove...

**Comment by schuettecarsten** [comment]
But this is a breaking change from the "old" HamburgerMenu behaviour. So at least, there should be an option for this.

**Comment by dg2k** [comment]
The change referred to is a recent one -- hence not the original design -- and I only noticed when I tested with the latest source code. If the current mode of operation is to be accepted, it totally negates the option for **CompactInline** display mode because you cannot maintain this mode.

I also noticed another subtle change with the latest revamp of HamburgerMenu.xaml (VisualStateManager in DataTemplate etc) and perhaps I'll post it as an issue on its own, but will mention it anyway as this relates to the issue raised here.

The issue is that NavButtons do not update background brush dynamically. Before the VisualStateManager was introduced into **NavButtonTemplate DataTemplate** all was fine but now **ContentPresenter.Background** doesn't update (when in VisualState "Normal").

There are other changes such as the left-margin 4px vertical line menu indicator -- is this to replace the NavButton highlight color, because the hightlight color doesn't seem to work always (still testing this).
